.github/ISSUE_TEMPLATE/chore.yml

```
@@ -14,23 +14,26 @@ body:
     id: what
     attributes:
       label: What should be done?
-      description: Additional context you would like to provide.
+    validations:
+      required: true
   - type: textarea
     id: where
     attributes:
       label: Where?
-      description: Include a permalink if possible.
+      description: Include permalinks if possible.
+    validations:
+      required: true
   - type: textarea
     id: dod
     attributes:
       label: Definition of Done
-      description: What should be done to consider this issue done?
+      description: What should be done to consider this issue done? List everything that applies.
       value: |
-        - [ ] unit tests added
-        - [ ] integration tests added
-        - [ ] compatibility tests added
-        - [ ] spot refactoring done
-        - [ ] documentation updated
-        - [ ] anything else?
+        - unit tests added;
+        - integration tests added;
+        - compatibility tests added;
+        - spot refactorings done;
+        - user documentation updated;
+        - something else?
```

As far as I understand English grammar, if bullet points are full sentences, they should start from capital letters - https://www.linkedin.com/pulse/how-capitalize-punctuate-bullet-points-l-michelle-baker-phd/.

```suggestion
        - Unit tests added.
        - Integration tests added.
        - Compatibility tests added.
        - Spot refactorings were done.
        - User documentation updated.
        - Something else?
```

---

Isn't that example B from that article rather than example D?

---

I'm not sure :) "Unit tests added" sounds like a complete sentence. But I'm ok with your proposal too.